composer/models.py

```diff
@@ -10,7 +10,7 @@
 from django.core.validators import MaxValueValidator, MinValueValidator
 from django.db import models
 from django.utils.translation import ugettext_lazy as _
-from jmbo.managers import DefaultManager, PermittedManager
+from .managers import DefaultManager, PermittedManager
```

Comment:
I believe the preferred way is to add the app name even if its a local relative import. so `from composer.managers`.